build/setup-kubectl.sh

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
 set -e
 
+DEFAULT_INSTALL_LOCATION="/app/kubectl/linux"
+DEFAULT_KUBECTL_VERSION="v1.21.2"
+
+# Arguments: kubectl version, install location
 function installKubectl {
-    apk update
-    apk upgrade
+    local loc_kube_version=$1
+    local loc_setup_location=$2
+
+    if [ -z "$loc_kube_version" ]; then
+        loc_kube_version=$DEFAULT_KUBECTL_VERSION
+    fi
+
+    if [ -z "$loc_setup_location" ]; then
+        loc_setup_location=$DEFAULT_INSTALL_LOCATION
+    fi
 
-    apk add -u kubectl --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+    echo "Setting up kubectl $loc_kube_version in $loc_setup_location"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/${loc_kube_version}/bin/linux/amd64/kubectl
```

Comment:
This will cause a problem if you are using ARM64 because it is only install the amd64 version

Reply:
@pkoelbl we tested in ARM64 Mac M1 max , we didn't face any errors/issues for both with and without routing. Tired couple of times connecting and disconnecting, it worked fine. Can you test this PR and provide the logs ? here is the CLI Version 1.0.20221129.4 and set BRIDGE_ENVIRONMENT='dev' in your environment variables. Thanks.

Reply:
Once you confirm we will proceed further, even with our testing team we didn't see any issues with M1 Mac's. I am wondering if I am missing something. Thanks.

Reply:
@pkoelbl any update on this ? At this moment this PR is mature enough to be merged into main. Please let us know. Thanks.